Tests/RegexTests/MatchTests.swift

```diff
@@ -831,5 +831,156 @@ extension RegexTests {
     matchTest(#"(?<a>.)(.)\k<a>"#, input: "abac", match: "aba", xfail: true)
     matchTest(#"\g'+2'(.)(.)"#, input: "abac", match: "aba", xfail: true)
   }
+  
+  // MARK: Character Semantics
+  
+  var eComposed: String { "é" }
+  var eDecomposed: String { "e\u{301}" }
+  
```

Comment:
- Try with both scalars being literals as well.
- Show group has a break requirement
- Show start/end of search may need boundary